docs/cue/reference/components.cue

```diff
@@ -879,6 +943,82 @@ components: {
 			}
 		}
 
+		env_vars: {
+			_http_proxy: {
+				description: """
+					The global URL to proxy HTTPS requests through.
+
+					If another HTTPS proxy is set in the configuration file or at a component level,
+					this one will be overridden.
+
+					The lowercase variant has priority over the uppercase one.
+					"""
+				type: string: {
+					default: null
+					examples: ["http://foo.bar:3128"]
+					syntax: "literal"
+				}
+			}
+			_https_proxy: {
```

Comment:
Can we add a `warning` here that redirecting otherwise insecure traffic to a non-secure proxy is a security risk. Do you think that's helpful?

Reply:
I guess, people putting in place a proxy are advanced enough to know it 🤔

Reply:
If you look at the doc from [AWS for example](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-proxy.html) the `HTTPS_PROXY` uses `http` and no `warning` section talks about it...
We can always update it if we see some issues about it.